app/supplejack/load/execution.rb

```diff
@@ -42,8 +42,6 @@ def call
 
     private
 
-    # TODO
-    # How can we reuse our existing request object?
```

Comment:
I feel like that's still a to do :/

Reply:
Yeah bro it is :(. I'll put it back to remind us